openapi/openapi-proposal.yaml

```diff
@@ -1831,11 +1849,11 @@ components:
       properties:
         code:
           type: number
-        message:
+        error:
           type: string
           description: >-
             `Failed to parse a tx.`
-        transaction:
+        extrinsic:
```

Comment:
Should this actually just go back to transaction?

Reply:
Yeah, for _submission_ we only submit _transactions._ But when it comes to _fetching,_ i.e. all the extrinsics from a block, then we will get a mix of inherents and transactions, and should therefore use _extrinsics._